cmd/consumer/consumer.go

```diff
@@ -86,9 +88,18 @@ func run() error {
 	}
 	consumer := consumer.NewKafkaConsumer(input)
 	go func() {
-		if err := consumer.InitSaramaConsumer(); err != nil {
+		err := wait.PollImmediateInfinite(500*time.Millisecond, func() (bool, error) {
+			if err := consumer.InitSaramaConsumer(); err != nil {
+				return false, nil
```

Comment:
Not sure whether it will affect condition of Poll, we may want to `return false, err` here

Reply:
If we return error, then we error out from the wait method. This behavior will be same as before.

Reply:
Got it. Thanks for clarifying